src/simperium/client.js

```diff
@@ -71,6 +71,10 @@ Client.prototype.bucket = function( name ) {
 
 	if ( this.open ) channel.onConnect();
 
+	bucket.hasLocalChanges = function() {
```

Comment:
The `Channel` constructor currently overrides the bucket methods related to the syncing status. I think we should move this definition of the bucket's `hasLocalChanges` there ([see source][src]).

Also add a default implementation the the Bucket itself that just runs the callback with `false`. At this point [the `Bucket` source][bucket] somewhat serves as an API definition.

[src]: https://github.com/Simperium/node-simperium/blob/aeef89dfd7502ad84deafc5178b0a954d1866414/src/simperium/channel.js#L296-L308
[bucket]: https://github.com/Simperium/node-simperium/blob/aeef89dfd7502ad84deafc5178b0a954d1866414/src/simperium/bucket.js

Reply:
Gotcha, updated in f06960f